models/wifi.lua

```diff
@@ -87,3 +87,17 @@ minetest.register_on_joinplayer(function(player)
 	local inv = player:get_inventory()
 	inv:set_size("more_chests:wifi", 8*4)
 end)
+
+minetest.register_allow_player_inventory_action(function(player, action, inventory, inventory_info)
+	if inventory_info.to_list == "more_chests:wifi" or inventory_info.from_list == "more_chests:wifi" then
+		local pos = player:get_pos()
+		local witem = player:get_wielded_item()
```

Comment:
`ItemStack:get_tool_capabilities()` would be more reliable to also take overrides into account.

Reply:
But `get_tool_capabilities()` doesn't return `range` value. Range of the item can't be overrided using meta(I tried to do it a while ago).

The one possible improvement I see now is using `witem:get_definition()` instead of retrieving definition from `minetest.registered_items`